File: 2023/java/src/test/java/org/frieder/aoc/day5/TestDay5A.java
package org.frieder.aoc.day5;

import org.frieder.aoc.day5.a.Solution5A;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay5A {
    @Test
    public void testExample() throws IOException {
        assertEquals(35D, Solution5A.getResult("../resources/5/test_a.txt"), 0);
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(196167384D, Solution5A.getResult("../resources/5/input.txt"), 0);
    }
}

File: 2023/php/test/TestDay5A.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day5\a\Solution5A;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay5A extends TestCase
{

    public function testExample(): void
    {
        $this->assertSame(35, Solution5A::getResult("../resources/5/test_a.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(196167384, Solution5A::getResult("../resources/5/input.txt", TestLogger::createTestLogger()));
    }

}